debian/postinst

```diff
@@ -5,17 +5,19 @@ set -e
 
 case "$1" in
   configure)
-    # get kolibri user from kolibri installer package:
-    db_get kolibri/user
-    KOLIBRI_USER="$RET"
-    KOLIBRI_DIR="$(getent passwd $KOLIBRI_USER | awk -F ':' '{print $6}')"/.kolibri
+    # get KOLIBRI_HOME from kolibri installer package:
+    source /etc/default/kolibri
     echo "
 location /static {
-    alias  $KOLIBRI_DIR/static;
+    alias  $KOLIBRI_HOME/static;
 }
  location /content {
-    alias   $KOLIBRI_DIR/content/;
-}"> /etc/kolibri/nginx_cache.conf
+    alias   $KOLIBRI_HOME/content/;
+}
+"> $KOLIBRI_HOME/nginx.conf
+
+    echo "include $KOLIBRI_HOME/nginx.conf;" > /etc/kolibri/nginx.d/099-user.conf
+
```

Comment:
@jredrejo @jamalex - this gives the user space back some control of the Nginx configuration while retaining the package's control of the defaults (perhaps `$KOLIBRI_HOME/nginx.conf` should change to `$KOLIBRI_HOME/var/nginx.conf` or similar, because it's not supposed to be edited by the user, but generated from `options.ini`).

I can also imagine that we have `$KOLIBRI_HOME/var/uwsgi_touch_reload` such that user space can touch the file, and then UWSGI reloads.

I would suggest a mechanism in Kolibri that simply writes the two `/content` and `/static` directives, it could be part of `kolibri.utils.conf` and simply write the file every time Kolibri starts.

I'll check out how in which order Nginx loads these files and whether the `location` directives going first or last take precedence :)